python/mxnet/gluon/block.py

```diff
@@ -1040,7 +1052,12 @@ def register_child(self, block, name=None):
         super(HybridBlock, self).register_child(block, name)
         self._clear_cached_op()
 
-    def hybridize(self, active=True, **kwargs):
+    def hybridize(self, active=True, backend=None, backend_args=None, **kwargs):
```

Comment:
hmmm. This is specific for hybridblock? Can we add documentation?

Reply:
Based on prior discussion with @samskalicky, documentation should describe what happens if input shapes change in subsequent forward calls. (Ie. currently no repartitioning is triggered).

Reply:
Added the description for hybridblock hybridize().

Reply:
Didn't see it - did you push?

Reply:
Pushed now.

Reply:
the concept of SubgraphBackendRegistry, PostPartition, etc are new and not very straightforward to users. Is it possible to also add a link to any tutorial that teaches user how to register a subgraph backend?

Reply:
As discussed offline, we plan to add a tutorial as part of our next PR and link it to the example. I have put together the TODO list for the next PR in this github issue #17532 .